<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmBaseSet</class>
 <widget class="QWidget" name="FrmBaseSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gbledset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1001</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>LED参数、测试项目设置</string>
   </property>
   <widget class="QComboBox" name="cbledtype">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>29</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>2脚1晶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4脚2晶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4脚3晶共阳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4脚3晶共阴</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6脚3晶</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>脚位类型：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>9</y>
      <width>341</width>
      <height>115</height>
     </rect>
    </property>
    <property name="title">
     <string>POL极性判断参数</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>延时(ms)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>电压(V)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>电流(mA)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电压上限(V)：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电压下限(V)：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="edtdelay">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>25</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtvf">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>56</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtma">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>88</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtvfmax">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>29</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edtvfmin">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>默认等外BIN（单项参数在上下限范围内，但不在有效分级参数内时默认BIN号）：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbinout">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>77</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tvtestitem1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tvtestitem2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tvtestitem3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tvtestitem4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
